src/harness/fourslash.ts

```diff
@@ -2878,8 +2866,8 @@ namespace FourSlash {
             }
             const actualContent = this.getFileContent(this.activeFile.fileName);
 
-            if (this.normalizeNewlines(actualContent) !== this.normalizeNewlines(expectedContent)) {
-                this.raiseError(`verifyFileAfterApplyingRefactors failed: expected:\n${expectedContent}\nactual:\n${actualContent}`);
+            if (actualContent !== expectedContent) {
+                this.raiseError(`verifyFileAfterApplyingRefactors failed: expected:\n${makeWhitespaceVisible(expectedContent)}\nactual:\n${makeWhitespaceVisible(actualContent)}`);

```

Comment:
This is going to be a pain because we'll be unable to copy+paste the actual back into the test. If we're going to do this, we should only do it when the files differ *only* in whitespace